cnxepub/formatters.py

```diff
@@ -95,15 +95,15 @@ def _generate_ids(self, document, content):
             'blockquote', 'q', 'code', 'pre', 'object', 'img', 'audio',
             'video',
             ]
-        elements_xpath = '|'.join(['.//{}|xhtml:{}'.format(elem, elem)
+        elements_xpath = '|'.join(['//{}|//xhtml:{}'.format(elem, elem)
```

Comment:
This change appears to generate ids for the top level `<div>`, but it's not reflected in the tests.  Are you sure this is correct?